zipline/assets/assets.py

```diff
@@ -91,6 +95,35 @@
 })
 
 SymbolOwnership = namedtuple('SymbolOwnership', 'start end sid symbol')
+OwnershipPeriod = namedtuple('OwnershipPeriod', 'start end sid value')
```

Comment:
Yeah I think we should consolidate these, since `OwnershipPeriod` is just a generalized version of `SymbolOwnership`.